src/incubator/TextField/Input.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import {TextInput, TextInputProps, StyleSheet, Platform} from 'react-native';
+import {ForwardRefInjectedProps} from '../../commons/new';
+import FieldContext from './FieldContext';
+
+export interface InputProps extends TextInputProps, React.ComponentPropsWithRef<typeof TextInput> {
+  hint?: string;
+}
+
+const Input = ({style, hint, forwardedRef, ...props}: InputProps & ForwardRefInjectedProps) => {
+  const context = useContext(FieldContext);
+  const placeholder = !context.isFocused ? props.placeholder : (hint || props.placeholder);
+  return (
+    <TextInput
+      style={[styles.input, style]}
+      {...props}
+      placeholder={placeholder}
+      ref={forwardedRef}
+      underlineColorAndroid="transparent"
+      accessibilityState={props.editable === false ? {disabled: true} : undefined}
```

Comment:
It used to be plural in rn59, [see here](https://reactnative.dev/docs/0.59/touchablewithoutfeedback#accessibilitystates)
Not it's singular, since rn61 [see here](https://reactnative.dev/docs/0.62/touchablewithoutfeedback#accessibilitystate)

Anyway, checked it and it reads the correct trait.

Reply:
But does it update if say it starts as disabled=true and then turned to disabled=false?

Reply:
Ok, I fixed it. Now it updates accordingly.